### PR TITLE
Show help message with no stack file and flags

### PR DESCRIPTION
**Pull request by yankeexe**
Signed-off-by: Yankee Maharjan <yankee.exe@gmail.com>

Addresses #894 

## Description

Show help message when no flags are passed (`--yaml` or `--from-store`) and no `stack.yml` file is present in the current working directory.

image

## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

Currently the CLI returns no result, confusing user what happened. Showing help message will guide the user to take corrective actions for the `generate` command.

## How Has This Been Tested?

run the command below with no flags and no `stack.yml` file in current working directory.

```bash 
go run . generate
```

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: commands/generate.go [hunk 1]
```diff
@@ -150,6 +151,9 @@ func runGenerate(cmd *cobra.Command, args []string) error {
 		if parsedServices != nil {
 			services = *parsedServices
 		}
+	} else {
+		fmt.Println("`stack.yml` file not found in the current directory.\nUse `--yaml` to pass a file or `--from-store` to generate using store image.")
+		os.Exit(0)
```
Comment (alexellis) [inline commands/generate.go:156]: In this instance can you exit 1? In the way, the program signals to its caller that there was an error.

**Review by alexellis — CHANGES_REQUESTED** [review]
Looks good so far, just one change needed

File: commands/generate.go (hunk 1 above)
Comment (yankeexe) [reply commands/generate.go:156]: okay!

**Comment by yankeexe** [comment]
@alexellis  I have update with status code 1 for the code. This is ready to merge. 
image

**Comment by alexellis** [comment]
"using the function store" instead of "using store image" would sound a little better 👍

**Comment by yankeexe** [comment]
> "using the function store" instead of "using store image" would sound a little better +1

@alexellis done :heavy_check_mark:

**Comment by alexellis** [comment]
Block quotes cannot be pasted and executed on bash. So you're better using a backtick for the whole message and double quotes inside it. Not the other way around. That will also mean you can remove the \n and just break the line yourself. Have a look at where else we have done this in the code.

Sorry to get you to make another change!

**Comment by yankeexe** [comment]
> Block quotes cannot be pasted and executed on bash. So you're better using a backtick for the whole message and double quotes inside it. Not the other way around. That will also mean you can remove the \n and just break the line yourself. Have a look at where else we have done this in the code.
> 
> Sorry to get you to make another change!

@alexellis TIL about that.  :smile:  I will send the change.

**Review by alexellis — APPROVED** [review]
Looking good :+1:

**Pull request merged**